sdk/src/main/java/com/hedera/hashgraph/sdk/Executable.java

```diff
@@ -86,6 +86,10 @@ public SdkRequestT setNodeAccountIds(List<AccountId> nodeAccountIds) {
     @Override
     @FunctionalExecutable
     public CompletableFuture<O> executeAsync(Client client) {
+        if (maxAttempts == Client.DEFAULT_MAX_ATTEMPTS) {
+            maxAttempts = client.getMaxAttempts();
+        }
```

Comment:
This way of handling maxAttempts seems odd to me.  If I understand correctly, the idea is that the user can set `maxAttempts` on the `Executable`, and then that new, non-default value will stick, instead of being set to `client.getMaxAttempts()` on execute.

However, if the user sets the `maxAttempts` on the `Executable` to 10 (the value of `Client.DEFAULT_MAX_ATTEMPTS`), their custom `maxAttempts` of 10 will be overwritten by `client.getMaxAttempts()`, which might not be 10 if the user has changed it.